backend/tests/v5/e2e/services/chat/modelEvents/modelSettings.test.js

```diff
@@ -223,9 +223,13 @@ const modelAddedTest = () => {
 				setTimeout(reject, 1000);
 			});
 
-			const payload = { name: ServiceHelper.generateRandomString(),
+			const payload = {
+				name: ServiceHelper.generateRandomString(),
 				type: generateRandomString(),
-				number: generateRandomString() };
+				number: generateRandomString(),
```

Comment:
Can we also check that on a calibration update event (it's probably in a different file), we're also triggering a model settings update event?